### PR TITLE
whishlist features of ecommerce project

### PR DESCRIPTION
**Pull request by devAmlan**
Wishlist feature of ecommerce app

1. Users can add a product to the wish list
2. Users can move to cart from wish list
3. Users can add to wish list from the cart
4. Used a wish list context and state to save my wish list product data

Preview ->
AnyConv com__mlan-store-wishlist

### CODE REVIEW

File: frontend/src/components/wishlistproduct/WishlistProduct.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                <h5>&#8377; {productprice}</h5>
+                <h5><s>&#8377; {originalprice}</s></h5>
+            </div>
+            <div className="ratings">
+                <div className="ratingbox">
+                    <h3>{productrating} <AiFillStar /></h3>
+                </div>
+            </div>
+            <div className="add_cart_btn">
+                <button onClick={() => { addtocartFromwishlist(product) }}>ADD TO CART</button>
```
Comment (rishabh486) [inline frontend/src/components/wishlistproduct/WishlistProduct.js:24]: You can make function make all camel case

File: frontend/src/context/WishlistContext.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        dispatch({ type: "ADD_TO_CART", payload: product })
+        setWishlistproducts(wishlistproducts.filter(item => item.productid !== product.productid))
+
+    }
+    const movetoWishlist = async (product) => {
+        setWishlistproducts([...wishlistproducts, product])
+        dispatch({ type: "REMOVE_FROM_CART", payload: product })
+    }
+    return (
+        <WishlistContext.Provider value={{ addtoWishlist, removefromWishlist, wishlistproducts, movetoWishlist, addtocartFromwishlist }}>
```
Comment (rishabh486) [inline frontend/src/context/WishlistContext.js:31]: Function name should be in camel case

**Review by rishabh486 — CHANGES_REQUESTED** [review]

**Review by DivyanshSareen — APPROVED** [review]
Clean code. Detailed commits and PR description. Good to merge

File: frontend/src/context/WishlistContext.js (hunk 2 above)
Comment (devAmlan) [reply frontend/src/context/WishlistContext.js:31]: Thanks for the suggestion @rishabh486  , I will change the name to camel case.

**Pull request merged**